docs/pages/guides/facebook-authentication.mdx

```diff
@@ -1,316 +1,63 @@
 ---
-title: Authentication with Facebook and AuthSession API
-sidebar_title: Using Facebook authentication
-description: A guide on setting up and using Facebook authentication with AuthSession API in your Expo app.
+title: Using Facebook authentication
+description: A guide on using react-native-fbsdk-next library to integrate Facebook authentication in your Expo project.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
-import { Tabs, Tab } from '~/ui/components/Tabs';
-import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
-import { Step } from '~/ui/components/Step';
 
-AuthSession API allows [browser-based authentication](/versions/latest/sdk/auth-session/#how-web-browser-based-authentication-flows-work) (using OAuth or OpenID Connect) to your project for Android, iOS, and the web. This guide provides steps on configuring and using the API with Facebook and a [development build](/develop/development-builds/create-a-build/).
+The [`react-native-fbsdk-next`](https://github.com/thebergamo/react-native-fbsdk-next/) library provides a wrapper around Facebook's Android and iOS SDKs. It allows integrating Facebook authentication into your Expo project and provide access to native components.
 
-<Step label="1">
+This guide provides additional information on configuring the library with Expo for Android.
 
-## Install libraries
+## Prerequisites
 
-To use AuthSession API, you'll need to install the following packages in your project:
+`react-native-fbsdk-next` requires [custom native code](/workflow/customizing/) and cannot be used with Expo Go. Before proceeding, you can either use:
 
-<Terminal cmd={['$ npx expo install expo-auth-session expo-crypto expo-web-browser']} />
+- Android Studio or Xcode to build your project locally (see [Prebuild](/workflow/prebuild/) for more information)
+- [Development builds](/develop/development-builds/create-a-build/)
```

Comment:
i think we can remove these links and the "Before proceeding, you can either use:"bit. we can saying that it requires custom native code.

maybe something like:

> The `react-native-fbsdk-next` library can't be used in the "Expo Go" app because it requires custom native code. [Learn more about adding custom native code to your app](https://docs.expo.dev/workflow/customizing/).